pyproject.toml

```
@@ -45,6 +45,12 @@
     name = "Contributor-facing changes"
     showcontent = true
 
+[build-system]
+build-backend = "setuptools.build_meta"
+requires = [
+  "setuptools>=45",
+  "setuptools_scm[toml]>=6.2",
```

Mind keeping these sparse?
```suggestion
  "setuptools >= 45",
  "setuptools_scm[toml] >= 6.2",
```

---

sparse in this context took me a minute.... :)

---

Teach me a better word and I'll use it instead :)